### ISSUE TITLE
Binary signing for Windows

### ISSUE THREAD

**Issue by hacdias** [ipfs-shipyard/ipfs-desktop#705]
Right now, on Windows, when trying to execute the binary, the user get this screen:

![image](https://user-images.githubusercontent.com/5447088/48933218-698d4980-eef7-11e8-9e64-171d3cf69ed8.png)

They will need to click on **More Info** and then a button **Run anyway** will show up, like this:

![image](https://user-images.githubusercontent.com/5447088/48933206-61350e80-eef7-11e8-9758-31a3992b0757.png)

What we want is not to have that screen showing up and look like a more authentic program. To do so, we will need to sign the releases binaries of IPFS Desktop.

There are some documents on Microsoft website:
- [**Cryptographic Tools**](https://docs.microsoft.com/en-us/windows/desktop/seccrypto/cryptography-tools)
- **[Sign Tool](https://docs.microsoft.com/en-us/windows/desktop/seccrypto/signtool)**

Basically we need a certificate and then use the `signtool` to sign the binary. This can be done automatically when there is a tag release via the CI. We still need to setup publishing on tags to GitHub releases.

Electron Builder supports auto-uploading to GitHub releases when there's a tag, but we need to have an environment variable with the GH API key. /cc @ipfs-shipyard/gui

**Comment by hacdias** [comment]
Binary signing is set up and working. The next version will have Windows signed binaries.
